app/serializers.py

```diff
@@ -0,0 +1,59 @@
+# TODO: Import the User from django.conf.settings.AUTH_USER_MODEL if the base user model is changed
+import re
+
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from .models import Currency, CurrencyPrice
+
+
+class CurrencyConverterSerializer(serializers.Serializer):
+    from_currency = serializers.CharField(max_length=3)
+    to_currency = serializers.CharField(max_length=3)
+    amount = serializers.FloatField(min_value=0)
+
+    def validate_from_currency(self, value):
+        if re.search(r'<.*?>', value):
```

Comment:
you should make it as a function and pass it to validators param in serializer to follow DRY #1